crates/bevy_scene/src/scene.rs

```diff
@@ -18,3 +25,68 @@ impl Scene {
         Self { world }
     }
 }
+
+impl Scene {
+    /// Clone the scene.
+    ///
+    /// This method will return a `SceneSpawnError` if either a type is not registered in the
+    /// provided `AppTypeRegistry` or doesn't reflect the `Component` trait.
+    pub fn clone(&self, type_registry: &AppTypeRegistry) -> Result<Scene, SceneSpawnError> {
+        let mut new_world = World::new();
+        new_world.insert_resource(type_registry.clone());
+        self.write_to_world(&mut new_world)?;
+        new_world.remove_resource::<AppTypeRegistry>();
```

Comment:
I don't understand the logic here, so the scene needs the type registry to write itself to the world, but not after that anymore? Doesn't that break any runtime logic on the new world?

Reply:
resources in scene are completely ignored. leaving it doesn't matter that much. anyway, removed that as per comment below